scripts/proto_gen_go.sh

```diff
@@ -50,7 +50,7 @@ mkdir -p planpb
 
 mkdir -p ../../cmd/tools/migration/legacy/legacypb
 
-protoc_opt="${protoc} --proto_path=${API_PROTO_DIR} --proto_path=/usr/local/include --proto_path=."
+protoc_opt="${protoc} --proto_path=${API_PROTO_DIR} --proto_path=/usr/local/include --proto_path=/usr/include --proto_path=."
```

Comment:
What is the purpose of adding this?

Reply:
After I install the `libprotobuf-dev`, `descriptor.proto` is located the newly added path on my local pc.